sdk/openai/Azure.AI.OpenAI/api/Azure.AI.OpenAI.netstandard2.0.cs

```diff
@@ -590,6 +591,8 @@ public abstract partial class ChatMessageContentItem : System.ClientModel.Primit
     public partial class ChatMessageImageContentItem : Azure.AI.OpenAI.ChatMessageContentItem, System.ClientModel.Primitives.IJsonModel<Azure.AI.OpenAI.ChatMessageImageContentItem>, System.ClientModel.Primitives.IPersistableModel<Azure.AI.OpenAI.ChatMessageImageContentItem>
     {
         public ChatMessageImageContentItem(Azure.AI.OpenAI.ChatMessageImageUrl imageUrl) { }
+        public ChatMessageImageContentItem(string imageDataUri) { }
```

Comment:
we have a guideline in .NET saying that URI should be represented by Syste.Uri, not string. It's fine to have string overloads as helpers (and so have that too)

Reply:
I would call the parameter simply imageUri, i.e. remove "data"

Reply:
I will rename the variable, but I have to ask, did you read my message for the PR? the whole point of this change is to use string over `System.Uri` because `System.Uri` is not usable because of it's limitation. this is not a helper for `System.Uri`. the api is already working with `System.Uri` and frankly this PR would not be needed, if `System.Uri` wasn't limited.

I say this, because I don't want any misunderstanding about why I am doing this 😊 I'd love to use `System.Uri` since it is the standard way in dotnet and eventually once `System.Uri` is fixed we can hopefully revert the changes here, but I don't see this happing sometime soon and even if, since `System.Uri` is part of the BCL it would take time until it would be GA.